.github/instructions/nx.instructions.md

```
@@ -28,5 +28,13 @@ If the user wants to generate something, use the following flow:
 - read the generator log file using the 'nx_read_generator_log' tool
 - use the information provided in the log file to answer the user's question or continue with what they were doing
 
+# Running Tasks Guidelines
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)